hometask6

```diff
@@ -0,0 +1,68 @@
+# Task 1
+#
+# import time
+# def my_decorator(func):
+#     def wrap():
+#         print(time.ctime())
+#         func()
+#         fname = func.__name__
+#         print(fname)
+#     return wrap
+#
+# @my_decorator
+# def my_func():
+#     print('This is my func')
+#
+# my_func()
+
+# Task 2
+# class MyException(Exception):
+#     pass
+# if True:
+#     raise MyException('Custom exception is occured')
+
+# Task 3
+
+# while True:
+#     try:
+#         class CtxManager:
+#             def __enter__(self):
+#                 print('==========')
+#                 # return "Hello, world!"
+#                 while True:
```

Comment:
I guess this part should be moved outside the context manager and be putted under its usage part:
```
with CtxManger():
    while True ...
        ...